container/postgres_conf_template.conf

```diff
@@ -0,0 +1,23 @@
+data_directory = '%POSTGRES_DATA_DIR%/%POSTGRES_NAME%'
+hba_file = '%POSTGRES_DATA_DIR%/%POSTGRES_NAME%/pg_hba.conf'
+ident_file = '/etc/postgresql/only/pg_ident.conf'
+external_pid_file = '/tmp/%POSTGRES_NAME%.pid'
+listen_addresses = '*'
+port = 5432
+max_connections = 100
+unix_socket_directories = '/tmp'
+ssl = off
+shared_buffers = 128MB
+dynamic_shared_memory_type = posix
+max_wal_size = 1GB
+min_wal_size = 80MB
+log_line_prefix = '%m [%p] %q%u@%d '
+log_timezone = 'Europe/Rome'
```

Comment:
While Rome is undoubtedly a very beautiful city, it would be better to stick with UTC here and below ;)

Reply:
I must admit, I selected the timezone without giving it much thought. Is there truly any alternative timezone available for CE(S)T, apart from that one? ;-P

By the way, I understand your perspective, and it does indeed make sense. I will adjust it as per your suggestion.